### PR TITLE
Modified consumers.go to make it easier to extend to new topics and i…

### PR DESCRIPTION
**Pull request by AaronH88**
…mplement new functions for each topic.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [X] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -138,8 +138,10 @@ func main() {
 	if cfg.KafkaConfig != nil {
 		log.Info("Starting Kafka Consumers")
 		go func() {
-			consumerService := services.NewKafkaConsumerService(cfg.KafkaConfig)
-			consumerService.Start()
+			playbookConsumer := services.NewKafkaConsumerService(cfg.KafkaConfig, "platform.playbook-dispatcher.runs")
+			playbookConsumer.Start()
+			platformInvConsumer := services.NewKafkaConsumerService(cfg.KafkaConfig, "platform.inventory.events")
```
Comment (biancarosa) [inline main.go:143]: You could split in two go func()'s just to make sure they are running separate, I'm pretty sure .Start is a blocking call ;)

File: main.go [hunk 2]
```diff
@@ -138,8 +138,10 @@ func main() {
 	if cfg.KafkaConfig != nil {
 		log.Info("Starting Kafka Consumers")
 		go func() {
```
Comment (biancarosa) [inline main.go:140]: You can remove that line now since there are two go funcs inside

Comment (AaronH88) [reply main.go]: Done :smile:

**Review by biancarosa — APPROVED** [review]
lgtm

**Pull request merged**